README.md

```diff
@@ -83,10 +83,9 @@ CONTAINERARCH=aarch64 ./build.sh platform debian-stretch
 
 ## Publishing a container
 
-1.  You need a Docker Cloud account with access to the `couchdbdev` organization. Ask the CouchDB PMC for assistance with this.
-2. `export DOCKER_ID_USER="username"`
```

Comment:
This needs to stay, for https://github.com/apache/couchdb-ci/blob/main/build.sh#L99

Otherwise +1

Reply:
Ah ok. I had just noticed it didn't seem t have any effect and it still asked for the login user